### PR TITLE
Seeding DB 

### PR DESCRIPTION
**Pull request by twentymurial33**
Seeding DB

### CODE REVIEW

File: prisma/seed.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const prisma = new PrismaClient();
+
+async function main() {
+  const insertResult = await prisma.food.createMany({
+    data: [
+      { id: 10, title: "toast", img: "bellablue", author: "lenovo" },
+      { id: 22, title: "sausage", img: "pinkpurple", author: "lenovo" },
+    ],
+  });
+  console.log(insertResult.count);
```
Comment (brentmclark) [inline prisma/seed.js:11]: Consider removing debugging code from code destined for the trunk.  Alternatively, if this isn't debugging code, consider making this more robust.  For example:

```suggestion
  console.info(`The prisma database was seeded with ${insertResult.count} items.`);
```

**Review by brentmclark — APPROVED** [review]
Looks great!

**Comment by brentmclark** [comment]
> Looks great!

I should add that I like the idea of seeding the database for now.  I could see it getting in the way eventually, though.  It might be worth moving this to a script that can be invoked manually instead of seeding the database on every launch.  Nothing to worry about right now, though.  I'm just planting a seed.

File: prisma/seed.js (hunk 1 above)
Comment (twentymurial33) [reply prisma/seed.js:11]: Ok will do moving forward...Have the habit of console.logging

**Pull request merged**